pom.xml

```diff
@@ -24,6 +24,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>3.0.0-beta2</version>
```

Comment:
do not use beta versions, they are unstable, and can have bugs or vulnurabilities